Gems/ImGui/Code/Include/ImGuiBus.h

```diff
@@ -87,7 +88,18 @@ namespace ImGui
         virtual void OverrideRenderWindowSize(uint32_t width, uint32_t height) = 0;
         virtual void RestoreRenderWindowSizeToDefault() = 0;
         virtual void ToggleThroughImGuiVisibleState() = 0;
+        virtual void SetDpiScalingFactor(float dpiScalingFactor) = 0;
+        virtual float GetDpiScalingFactor() const = 0;
         virtual void Render() = 0;
+
+        using ImGuiSetEnabledEvent = AZ::Event<bool>;
+        ImGuiSetEnabledEvent m_setEnabledEvent;
+
+        // interface
+        void ConnectImGuiSetEnabledChangedHander(ImGuiSetEnabledEvent::Handler& handler)
+        {
+            handler.Connect(m_setEnabledEvent);
+        }
```

Comment:
There is a Typo in ConnectImGuiSetEnabledChanged**Hander**

Reply:
Also since the m_setEnabledEvent is part of the public interface, having a function that only performs a connection to the event isn't gaining users much.
The user would able to just directly access the m_setEnabledEvent directly to call `Connect` instead.
It wouldn't be anymore lines for the users.

Instead I think this function should simplify registration with a callable object.
```suggestion
        using ImGuiSetEnabledEventHandler = AZ::EventHandler<bool>;
        using SetEnabledCallback = AZStd::function<void(bool)>;
        void ConnectImGuiSetEnabledChangedHander(SetEnabledCallback callback)
        {
                ImGuiSetEnabledEventHandler imGuiHandler{ AZStd::move(callback) };
                imGuiHandler.Connect(m_setEnabledEvent);
                return imGuiHandler;
        }
```

Reply:
Fix one misspelling - add a new one! Fixed.